server/api/users.js

```diff
@@ -126,15 +130,24 @@ internals.applyRoutes = function (server, next) {
     }
   });
 
-
   server.route({
     method: 'GET',
     path: '/users',
     config: {
       auth: {
-        strategies: ['simple', 'jwt', 'session'],
-        scope: ['root', 'admin', 'researcher']
+        strategies: ['simple', 'jwt', 'session']
       },
+      pre: [
+        {
+          assign: 'Authorization',
+          method: (request, reply) => {
+
+            Authorization(request, PermissionConfigTable['GET/api/users']) ?
+              reply(true) :
+              reply(Boom.conflict('Insufficient Authorization for user: ' + request.auth.credentials.user._id));
+          }
+        }
+      ],
```

Comment:
why have you added an extra authorization function to these routes. please remove all of them.